build.gradle

```diff
@@ -318,6 +328,7 @@ task renameCrowdin(type: Copy, dependsOn: ['crowdin', 'cleanCrowdin']){
 import groovy.json.JsonOutput
 import groovy.json.JsonSlurper
 import groovy.util.XmlSlurper
+import org.kohsuke.github.GHReleaseBuilder
```

Comment:
Please dont make unrelated changes.

Reply:
Hmm weird I do not remember adding that.